roles/nova-common/templates/custom-filters/custom_disk_filter.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2012 OpenStack Foundation
```

Comment:
shouldn;t this file be deleted from this PR given is now part of bbc-openstack-plugins